unified-codes/apps/data-service/src/app/types/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './DGraphDataSource';
```

Comment:
I think this might need to be `'./DgraphDataSource'` (lower case G).

It's weird because this transpiles fine on my local machine but the CI build didn't like it.

Reply:
81a91811d340fd6583b57e01c020f6582967623e should do it!